shared/getMetrics.js

```diff
@@ -191,7 +191,7 @@ const getMetrics = ({
         assertionsCount: requiredAssertionsCount,
         assertionsPassedCount: requiredAssertionsPassedCount,
         assertionsFailedCount: requiredAssertionsFailedCount
-    } = calculateAssertionPriorityCounts(result, 'REQUIRED');
+    } = calculateAssertionPriorityCounts(result, 'MUST');
```

Comment:
So with this change, because https://github.com/w3c/aria-at-app/blob/3808ed7f46642564718978aabd2765dd96c748c3/shared/getMetrics.js#L46 uses the priority string in combination with what has been named through the client's various `queries.js` files, it will incorrectly display the expected values for anywhere `TestPlanResultsTable` is getting used.

ie. [`queries.js`](https://github.com/w3c/aria-at-app/blob/3808ed7f46642564718978aabd2765dd96c748c3/client/components/TestRun/queries.js#L39) provides `requiredAssertionResults` but this change to `getMetrics` is expecting `mustRequiredAssertionResults`.

The following screenshot also shows that for each command, there are 2+ assertions but the command's assertion level status only shows `2 passed`, which would then also affect the total calculation for the entire test:
![screenshot showing inaccurately calculated numbers](https://github.com/w3c/aria-at-app/assets/7191577/84b88a79-7195-449e-83a2-176dee674de9)

2 things I think could happen here:
1. Revert the change to `getMetrics` until updating each of the named `(required|optional)AssertionResults` instances. But this may cause some confusion until then.
2. Update the named instances of `(required|optional)AssertionResults` as appropriate which means renaming the value being exported from `getMetrics` and also adding a migration to update metrics across all the TestPlanReports

Reply:
Hey @howard-e all of the changes have been made and a migration created.

Reply:
Great! Thanks for going the route of #2 as well. Definitely makes things cleaner for the future!